### ISSUE TITLE
NFC: (Bus card) Cant be emulated. Only scanned.

### ISSUE THREAD

**Issue by Killer74-hub** [flipperdevices/flipperzero-firmware#3454]
![Screenshot-20240216-175623](https://github.com/flipperdevices/flipperzero-firmware/assets/126610656/58514f04-9ddb-4e60-8acf-d1c7375a3b22)
![Screenshot-20240216-175616](https://github.com/flipperdevices/flipperzero-firmware/assets/126610656/baa86ea3-47a8-4ad2-b3bb-ab4de19b892b)
### Describe the bug.

The Card is this card: https://www.rmv.de/c/en/tickets/your-ticket/tickets-overview/annual-season-tickets

It is a german Bus ticket.

### Reproduction

1. Clone Card
2. Save the Card/Emulate Directly
3. Emulate the card

### Target

German Bus Card(NFC)

### Logs

```Text
Filetype: Flipper NFC device
Version: 4
# Device type can be ISO14443-3A, ISO14443-3B, ISO14443-4A, ISO14443-4B, ISO15693-3, FeliCa, NTAG/Ultralight, Mifare Classic, Mifare DESFire, SLIX, ST25TB
Device type: ISO14443-4A



scanned: 


72249131 [D][NfcScanner] Found 4 children
72249188 [D][Nfc] FWT Timeout
72249190 [D][Nfc] FWT Timeout
72249224 [D][Nfc] FWT Timeout
72249257 [D][Iso14443_4aPoller] Read ATS success
72249278 [I][NfcScanner] Detected 1 protocols
72249413 [D][Iso14443_4aPoller] Read ATS success
72249647 [D][DolphinState] icounter 555, butthurt 0


(I will give you the rest through discord if you need it. contact me.)
```

### Anything else?

Contact me through Discord if you want the NFC Card number/ more screenshots. I don´t want to give it to the public. 

discord: sniper74

**Comment by mxcdoam** [comment]
It seems like f0 doesn't support writing operations on emulated card. So it will only emulate "frozen" card state, the reader will be unable to update emulated card's memory and the transaction will fail. 
By the way, this also affects intercoms in my area: reader writes new data on each pass, if it is unable to write - key is declined.

**Comment by Killer74-hub** [comment]
> It seems like f0 doesn't support writing operations on emulated card. So it will only emulate "frozen" card state, the reader will be unable to update emulated card's memory and the transaction will fail. By the way, this also affects intercoms in my area: reader writes new data on each pass, if it is unable to write - key is declined.

can it be fixed by an firmware update? because some guys here said its broken with newer firmware:https://forum.flipper.net/t/flipper-zero-can-read-cards-but-cant-emulate-them/20006

**Comment by mxcdoam** [comment]
It might be not broken, but intended that way. Storing backups of transit passes on a pocket device might sound like a nice idea, but it usually goes against most transit authorities rules of service. As for the intercom keys - I am curious too if this is possible or not.

**Comment by Killer74-hub** [comment]
> It might be not broken, but intended that way. Storing backups of transit passes on a pocket device might sound like a nice idea, but it usually goes against most transit authorities rules of service. As for the intercom keys - I am curious too if this is possible or not.

Yeah sure. I mean, once its expired, It will also not work anymore. So it doesn´t matter.

**Comment by skotopes** [comment]
From all possible information you gave us the most useless. What card type was it?

**Comment by Killer74-hub** [comment]
> From all possible information you gave us the most useless. What card type was it?

ISO 14443-4A (Unknown)

UID: 0E 11 3B 11 78 0D 03

Historical Bytes:

RAW: 5A 43 56 44 56

ISO14443-3A data
ATOA: 00 48 SAK: 20



Let me know, what you want next. I Don´t know if i´ts a mifare or so. I am new in tghe flipper zero community. What do you still need?



(sorry for my bad english)

**Comment by skotopes** [comment]
Do you have proxmark?

**Comment by Killer74-hub** [comment]
> Do you have proxmark?

No

**Comment by Killer74-hub** [comment]
> Do you have proxmark?

Do you need something else?

**Comment by Killer74-hub** [comment]
https://www.eticket-deutschland.de/en/

maybe this helps. They do have this symbol.

**Comment by skotopes** [comment]
@Killer74-hub try to use any other tool you can find. For example mobile phone apps for nfc or any other reader. They may be using different detection technic and give some hints on card type.

**Comment by Killer74-hub** [comment]
I found it. it´s a: Mifare SmartMX I think

**Comment by Killer74-hub** [comment]
> @Killer74-hub try to use any other tool you can find. For example mobile phone apps for nfc or any other reader. They may be using different detection technic and give some hints on card type.

https://oliver-weikert.de/smart-cards/
![image](https://github.com/flipperdevices/flipperzero-firmware/assets/126610656/f2152a20-ca42-4119-a8f5-5a1a3380faad)

**Comment by Killer74-hub** [comment]
> @Killer74-hub try to use any other tool you can find. For example mobile phone apps for nfc or any other reader. They may be using different detection technic and give some hints on card type.

I have NFC Tools. I will try it right now

**Comment by skotopes** [comment]
We don’t support smarts yet. We will take a look as soon as get some systems with it in our hands.

**Comment by Killer74-hub** [comment]
> We don’t support smarts yet. We will take a look as soon as get some systems with it in our hands.

wait. Its not I think. My app says something different

**Comment by Killer74-hub** [comment]
Can I send it to you on Discord? Or is it safe for here?

**Comment by skotopes** [comment]
Yes you can find me on discord, search for skotopes.

**Comment by skotopes** [comment]
But keep in mind that @gornekich is nfc maintainer and you probably want to talk with him instead.

**Comment by Killer74-hub** [comment]
oh ok. I´ve sended it to you. Does @gornekich also have discord?

**Comment by skotopes** [comment]
Yes, but please ping him on Monday. He deserves some weekends.

**Comment by Killer74-hub** [comment]
> Yes, but please ping him on Monday. He deserves some weekends.

Alright. np. I cannot find him on Discord. what is his username?

**Comment by skotopes** [comment]
He’ll tell it on Monday.

**Comment by Killer74-hub** [comment]
> He’ll tell it on Monday.

Ok. Thank´s for your support here. I appreciate it!

**Comment by gornekich** [comment]
Hey @Killer74-hub . Thanks for information. We will get this card and start working on its support. However, I read datasheet and it's look like it is a well secured card, so flipper will read open data but full emulation might be impossible.

**Comment by Killer74-hub** [comment]
> Hey @Killer74-hub . Thanks for information. We will get this card and start working on its support. However, I read datasheet and it's look like it is a well secured card, so flipper will read open data but full emulation might be impossible.

Hello! Thank´s for your reply @gornekich . It might be a Mifare SmartMx. But I don´t think so. Could I send you the screenshot that I have sended @skotopes on discord? If so, what is your discord Name?

**Comment by gornekich** [comment]
> > Hey @Killer74-hub . Thanks for information. We will get this card and start working on its support. However, I read datasheet and it's look like it is a well secured card, so flipper will read open data but full emulation might be impossible.
> 
> Hello! Thank´s for your reply @gornekich . It might be a Mifare SmartMx. But I don´t think so. Could I send you the screenshot that I have sended @skotopes on discord? If so, what is your discord Name?

Sure. My nick is gornekich

**Comment by Killer74-hub** [comment]
> > > Hey @Killer74-hub . Thanks for information. We will get this card and start working on its support. However, I read datasheet and it's look like it is a well secured card, so flipper will read open data but full emulation might be impossible.
> > 
> > 
> > Hello! Thank´s for your reply @gornekich . It might be a Mifare SmartMx. But I don´t think so. Could I send you the screenshot that I have sended @skotopes on discord? If so, what is your discord Name?
> 
> Sure. My nick is gornekich

sended you a Friend request because I could not find you

**Comment by Killer74-hub** [comment]
> > > Hey @Killer74-hub . Thanks for information. We will get this card and start working on its support. However, I read datasheet and it's look like it is a well secured card, so flipper will read open data but full emulation might be impossible.
> > 
> > 
> > Hello! Thank´s for your reply @gornekich . It might be a Mifare SmartMx. But I don´t think so. Could I send you the screenshot that I have sended @skotopes on discord? If so, what is your discord Name?
> 
> Sure. My nick is gornekich



Hello,

something new?

**Comment by gornekich** [comment]
Still trying to get this card. Please, don't expect support for this card in nearest future

**Comment by Killer74-hub** [comment]
> Still trying to get this card. Please, don't expect support for this card in nearest future

Okay. I am waiting. Thanks:D!

**Comment by AdminEvilDRagon** [comment]
> Hey @Killer74-hub . Thanks for information. We will get this card and start working on its support. However, I read datasheet and it's look like it is a well secured card, so flipper will read open data but full emulation might be impossible.

Yo is it possible now, to use it? Not only scanning?

**Comment by Killer74-hub** [comment]
Still Waiting. Keep me Up to Date please If you got it:)
